src/main/resources/application.yml

```
@@ -11,9 +11,15 @@ spring:
 
-# JWT 설정
-jwt:
-  secret: ${JWT_SECRET:your-secret-key-here-must-be-at-least-256-bits-long-for-security}
-  access-token-lifetime: ${JWT_ACCESS_TOKEN_LIFETIME:1800000}  # 30분 (30 * 60 * 1000)
-  refresh-token-lifetime: ${JWT_REFRESH_TOKEN_LIFETIME:259200000}  # 3일 (3 * 24 * 60 * 60 * 1000)
\ No newline at end of file
+# 애플리케이션 설정
+app:
+  batch:
+    aimodel-index:
+      enabled: true  # AIModel 인덱싱 스케줄러 활성화 (증분 처리)
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)